src/domain/events/upcomingEventsSection/UpcomingEventsSection.tsx

```diff
@@ -43,7 +55,8 @@ export default function UpcomingEventsSection({ linkedId, keywords }: Props) {
       },
       first: 6,
     },
-    skip: !process.browser,
+    skip: !process.browser || defer,
+    ssr: false,
```

Comment:
```ts
  if (
    (context.renderPromises || client.disableNetworkFetches) &&
    options?.ssr === false
  ) {
    // If SSR has been explicitly disabled, and this function has been called
    // on the server side, return the default loading state.
    result = ref.current.result = {
      loading: true,
      data: void 0 as unknown as TData,
      error: void 0,
      networkStatus: NetworkStatus.loading,
    };
  } else if (options?.skip || options?.fetchPolicy === 'standby') {
    // When skipping a query (ie. we're not querying for data but still want to
    // render children), make sure the `data` is cleared out and `loading` is
    // set to `false` (since we aren't loading anything).
    //
    // NOTE: We no longer think this is the correct behavior. Skipping should
    // not automatically set `data` to `undefined`, but instead leave the
    // previous data in place. In other words, skipping should not mandate that
    // previously received data is all of a sudden removed. Unfortunately,
    // changing this is breaking, so we'll have to wait until Apollo Client 4.0
    // to address this.
    result = {
      loading: false,
      data: void 0 as unknown as TData,
      error: void 0,
      networkStatus: NetworkStatus.ready,
    };
  }
```
https://github.com/apollographql/apollo-client/blob/383ead2b098373ad4da33f6a53e0a9568c226a3c/src/react/hooks/useQuery.ts#L264-L293

Seems like the end result would be the same if we dropped `!process.browser`. I'll do that.